pixl_dcmd/tests/test_dicom_validator.py

```diff
@@ -52,16 +64,18 @@ def test_validation_passes_for_non_compliant_dicom(non_compliant_dicom_image) ->
     assert not validator.validate_anonymised(non_compliant_dicom_image)
 
 
-def test_validation_fails_after_invalid_tag_modification(vanilla_dicom_image) -> None:
+def test_validation_fails_after_invalid_tag_modification(
+    vanilla_single_dicom_image_DX,
+) -> None:
```

Comment:
should this test be using the `non_compliant_dicom_image` fixture so we're not duplicating the deletion of the patient name?

Reply:
Make sense to remove duplicated lines but `non_compliant_dicom_image` has already a deleted patient which validation_result is an empyt dict_keys() when doing:
```
validator.validate_original(non_compliant_dicom_image)
validation_result = validator.validate_anonymised(non_compliant_dicom_image)
```
so delete patient after `validator.validate_original` works
```
validator.validate_original(vanilla_single_dicom_image_DX)
del vanilla_single_dicom_image_DX.PatientName
validation_result = validator.validate_anonymised(vanilla_single_dicom_image_DX)
```
Do you have other suggestions? Let's catch up tomorrow for clarifications.

Reply:
ah yeah okay that's a fair point, ignore me!